static/assets/css/sections/home.css

```diff
@@ -18,6 +18,19 @@
   background-size: cover;
 }
 
+/*
+  Resolves https://github.com/hossainemruz/toha/issues/70
```

Comment:
I can remove this comment if you'd prefer to rely on git history for this type of thing. I just typically like to add comments around "magic css" which is in place for a unique/special problem

Reply:
I think keeping this comment will be helpful.